testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/infinispan/cache/AddDistributedCacheOnlineTest.java

```diff
@@ -17,6 +17,7 @@
 
 import java.io.IOException;
 import java.util.UUID;
+import java.util.concurrent.TimeoutException;
```

Comment:
Right, that's a leftover from my feeble previous attempt where I called `reload`, I'll get rid of that.